introduction.md

```diff
@@ -1,16 +1,16 @@
 hi, this is jitendra.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?